cmd/languageserver.go

```diff
@@ -29,34 +28,23 @@ func init() {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
-			exe, err := os.Executable()
-			if err != nil {
+			if exe, err := os.Executable(); err != nil {
```

Comment:
Until recently, we used to have noinlineerr in regal

https://github.com/open-policy-agent/regal/blame/27d19e6f72a07c9cc94ce7e50564268543fed2b4/.golangci.yaml#L24

I personally preferred that but just my 2c

Reply:
Mostly I was just checking in as we did have it for a long time in the project and I grew used to it I suppose.

I guess I can see the scope benefit too.

But personally find the more verbose err := foo()\nif err != nil to be easier to read, particularly when foo() is long with more params etc.

No strong feelings.